internal/xcrd/schemas.go

```diff
@@ -318,6 +318,31 @@ func CompositeResourceClaimSpecProps() map[string]extv1.JSONSchemaProps {
 	}
 }
 
+// CompositeResourceClaimStatusProps is a partial OpenAPIV3Schema for the status
+// fields that Crossplane expects to be present for all defined or published
+// infrastructure resources.
+func CompositeResourceClaimStatusProps() map[string]extv1.JSONSchemaProps {
```

Comment:
Is it related to the new policy?

Reply:
Yes, Since Mergo doesn't provide a way to merge deleting empty field, it needs an other way to sync status.
what I tried is extract a conditions field on srcComposite and the fields to be synced on dstClaim. and merge them into empty claim.
So that function is for getting a conditions field from srcComposite in here (internal/controller/apiextensions/claim/configurator.go).